discussions/models.py

```diff
@@ -0,0 +1,17 @@
+"""
+Models for user profile
+"""
+from django.conf import settings
+from django.db import models
+
+
+class DiscussionUser(models.Model):
+    """
+    Tracks the user's discussion user and profile
+    """
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='discussion_users')
+    username = models.CharField(max_length=26, null=True)
```

Comment:
Username comes from `open-discussions` [here](https://github.com/mitodl/open-discussions/blob/master/profiles/serializers.py#L25), so this will be `None` between the time when the record is created and the API call to create the user in `open-discussions` is complete.